CloudyCar/app/src/main/java/com/cloudycrew/cloudycar/createrequest/CreateRequestController.java

```diff
@@ -1,11 +1,21 @@
 package com.cloudycrew.cloudycar.createrequest;
 
 import com.cloudycrew.cloudycar.ViewController;
+import com.cloudycrew.cloudycar.controllers.RequestController;
+import com.cloudycrew.cloudycar.models.Route;
+import com.cloudycrew.cloudycar.models.requests.PendingRequest;
 
 /**
  * Created by George on 2016-11-05.
  */
 
 public class CreateRequestController extends ViewController<ICreateRequestView> {
-
+    private RequestController requestController;
+    public CreateRequestController(RequestController requestController) {
+        this.requestController = requestController;
+    }
+    public void saveRequest(Route userRoute, double price) {
+        PendingRequest newRequest = new PendingRequest(null,userRoute,price);
```

Comment:
Rather than creating a new request and setting the rider to null, would't it make more sense to pass in the route and price to the controller, and have the controller generate the request since it knows who it's making it for?